ts_src/psetv2/blinder.ts

```diff
@@ -164,10 +164,11 @@ export class Blinder {
           value,
           targetOutput.asset!,
           valueCommitment,
+          assetCommitment,
```

Comment:
the order of parameters changed, looks like a breaking change to me. if was intentional I suggest tagging v3.x.x so we make evident the signature change